container-builder/common/docker.go

```diff
@@ -138,13 +140,13 @@ func (d Docker) RemoveImagesFiltered(repo string, tag string) (bool, error) {
 
 	images, err := d.GetImages(types.ImageListOptions{Filters: filters})
 	if err != nil {
-		log.Errorf(err, "%s\n", err)
+		klog.V(log.E).Infof("%s\n", err)
 		return false, err
 	}
 	for _, image := range images {
 		item, err := d.Connection.ImageRemove(context.Background(), image.ID, types.ImageRemoveOptions{PruneChildren: true, Force: true})
 		if err != nil {
-			log.Errorf(err, "%s %s\n", err, item)
+			klog.V(log.E).Infof("%s %s\n", err, item)
```

Comment:
Maybe we should use `ErrorS` instead? https://github.com/kubernetes/klog/blob/main/klog.go#L1464